content/docs/ui/account-and-settings/verifying-your-account.md

```diff
@@ -11,42 +11,43 @@ navigation:
+The time it takes to completely unlock your account can vary based on how quickly you complete the required steps. The faster you complete the steps, the faster you can start sending!
 
-</call-out>
+### Unlock your Free Account
 
-## Verifying your Account
+In order to send email with your free Twilio SendGrid account, you must create and verify a Single Sender. Adding a Sender Identity will help protect your sending reputation and legitimize your account in the eyes of the inbox providers for better deliverability. For more information, see [Single Sender Verification]({{root_url}}/ui/sending-email/sender-verification/).
 
-The time it takes to completely unlock your account can vary based on the number of steps you are required to complete. The faster you complete the steps, the faster you can start sending!
+### Unlock your Paid Account
 
-*To verify your account:*
+1. Confirm your email address. If you need to resend the verification email, click **Resend account verification**. If you entered your email address incorrectly in the initial signup form, you can correct the email address and resend the verification email to the correct address. 
+1. Set up [Domain Authentication]({{root_url}}/ui/account-and-settings/how-to-set-up-domain-authentication/). Authenticating your domain shows email providers that Twilio SendGrid has your permission to send emails on your behalf. 
 
-Complete one or more of the following:
+<call-out>
+
+If you don't have access to your domain's DNS records, you can use the **Send To A Coworker** feature. This feature allows you to enter a colleague's email address so Twilio SendGrid can send them all the information they need to complete domain authentication on your behalf. 
 
-* Verify your email address
-* Set up [two-factor authentication]({{root_url}}/ui/account-and-settings/two-factor-authentication/#setting-up-two-factor-authentication)
-* Set up [domain authentication]({{root_url}}/ui/account-and-settings/how-to-set-up-domain-authentication/)
+![]({{root_url}}/img/<domain-auth-send-to-coworker>.png "Send to a coworker tab")
 
-You may need to complete one or more of these steps to completely verify your account. You cannot skip any of the verification steps, or your account won't be completely unlocked.
+</call-out>
 
-### Resend the verification email
+2. Set up [two-factor authentication]({{root_url}}/ui/account-and-settings/two-factor-authentication/#setting-up-two-factor-authentication). Two-factor authentication allows you to make your SendGrid account more secure by requiring authentication beyond a simple username and password when logging in.
+3. Add a [payment method]({{root_url}}/ui/account-and-settings/billing/#payment-methods). Your card will not be charged until your first bill is due. 
```

Comment:
I don't think this is accurate bc they have to reenter their credit card, I pinged Scott in the slack channel to confirm.